acceptance/framework/k8s/deploy.go

```diff
@@ -16,8 +16,6 @@ import (
 	"k8s.io/apimachinery/pkg/util/yaml"
 )
 
```

Comment:
Changed the function signature for CheckStaticServer* to allow passing in an `app` which is the app you're connecting from, and an `expectedSuccessOutput` which is the expected message back from the server. I needed to be able to make connections from the multiport app and not just static client, and each of the multiport services had a different success output.